src/data.js

```diff
@@ -8,7 +8,7 @@ module.exports = {};
  *
  */
 module.exports.Build = Entity.configure({
-  version: 1,
+  version: 2,
```

Comment:
Upgrading the version requires that both versions 1 and 2 be in the source code -- the whole `.configure({..})` block needs to be copied.  So

```js
module.exports.Build = Entity.configure({
  version: 1,
  ...
}).configure({
  version: 2,
  ...
  installationId: Entity.types.Number,
  ...
});
```